apps/dotcom/client/e2e/fixtures/Sidebar.ts

```diff
@@ -79,6 +79,7 @@ export class Sidebar {
 	async setLanguage(languageButtonText: string, language: string) {
 		await this.openLanguageMenu(languageButtonText)
 		await this.page.getByRole('menuitemcheckbox', { name: language }).click()
+		await this.page.keyboard.press('Escape')
```

Comment:
started seeing the language select test fail because it wasn't dismissing the menu before trying to open it again